easybuild/easyblocks/boostcray.py

```diff
@@ -0,0 +1,222 @@
+##
```

Comment:
@teojgo, I do not understand the existence of this file...
Why is it here? None of this recipes use boost... There is no boost recipe...
Is it just to make it Python3 compatible?

Reply:
Yeah I need to add boost as well for 20.08, version 1.67

Reply:
@victorusu Boost is used by SerialBox

Reply:
@victorusu , boostcray easyblock is needed, else COSMO does not build - tried several times to remove it, but it is required.